File: src/very_easy/SecondLargest.swift
func secondLargest(_ arr: [Int]) -> Int {
  let nums = Set(arr)
  var first = Int.min
  var second = Int.min
  for num in nums {
    if first < num {
      second = first
      first = num
    } else if second < num {
      second = num
    }
  }
  return second
}

File: src/very_easy/second-largest.ts
export const secondLargest = (nums: number[]): number => {
	const arr = new Set(nums);
	let first = Number.MIN_SAFE_INTEGER
	let second = Number.MIN_SAFE_INTEGER
	for(let num of arr) {
		if(first < num) {
			second = first
			first = num
		} if(second < num && first > second) {
			second = num
		}
	}
	return second
}
